frontend/src/components/DeploymentDialog.vue

```diff
@@ -76,12 +76,13 @@ export default class DeploymentDialog extends Vue {
     }
 
     pushAutoScroll (...output) {
-        var con = document.getElementsByClassName('content scrolling')[0];
-        if (con.scrollTop >= con.scrollHeight - con.clientHeight) {
-            this.output.push(...output);
+        const con = this.$refs.scrollBox.$el;
+        const scrollFuzz = 5; // allowable distance from bottom in pixels to enable autoscroll
```

Comment:
When you add a comment to explain the name, the name is probably poor.
Let's rename it to something better.

`autoScrollSnapMargin` maybe?